### PR TITLE
[Feature][Flink] Support multiple tables read and write

### PR DESCRIPTION
**Pull request by PeppaPage**
### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Please follow the guide to open ci on your fork repository. https://github.com/apache/seatunnel/runs/25166667347

File: seatunnel-translation/seatunnel-translation-flink/seatunnel-translation-flink-common/src/main/java/org/apache/seatunnel/translation/flink/sink/FlinkSink.java [hunk 1]
```diff
@@ -91,17 +89,30 @@ public Optional<GlobalCommitter<CommitWrapper<CommT>, GlobalCommT>> createGlobal
...
-        return sink.getCommitInfoSerializer().map(CommitWrapperSerializer::new);
+        if (checkForCommitter())
+            return sink.getCommitInfoSerializer().map(CommitWrapperSerializer::new);
+        else return Optional.empty();
     }
 
     @Override
     public Optional<SimpleVersionedSerializer<GlobalCommT>> getGlobalCommittableSerializer() {
-        return sink.getAggregatedCommitInfoSerializer().map(FlinkSimpleVersionedSerializer::new);
+        if (checkForCommitter())
```
Comment (litiliu) [inline seatunnel-translation/seatunnel-translation-flink/seatunnel-translation-flink-common/src/main/java/org/apache/seatunnel/translation/flink/sink/FlinkSink.java]: there should be a `{` behind the if clause, it makes reading easier.

File: seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java [hunk 2]
```diff
@@ -171,4 +191,40 @@ public SeaTunnelSink fallbackCreateSink(
         source.prepare(pluginConfig);
         return source;
     }
+
+    // catalogName.databaseName.[schemeName].tableName -> databaseName.[schemeName].tableName
+    public String extractTableIdName(TableIdentifier tableIdentifier) {
```
Comment (litiliu) [inline seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java]: is it better to add ut to cover this logic?

File: seatunnel-core/seatunnel-flink-starter/seatunnel-flink-starter-common/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java [hunk 3]
```diff
@@ -175,4 +194,40 @@ public SeaTunnelSink fallbackCreateSink(
...
+            tableId.append(tableIdentifier.getSchemaName());
+        }
+        if (tableIdentifier.getTableName() != null) {
+            if (tableId.length() != 0) {
+                tableId.append(".");
+            }
+            tableId.append(tableIdentifier.getTableName());
+        }
+        return tableId.toString();
+    }
```
Comment (litiliu) [inline seatunnel-core/seatunnel-flink-starter/seatunnel-flink-starter-common/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java]: duplicate with the method in the seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java ?

**Review by litiliu — CHANGES_REQUESTED** [review]

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-elasticsearch-e2e/src/test/java/org/apache/seatunnel/e2e/connector/elasticsearch/ElasticsearchIT.java [hunk 4]
```diff
@@ -203,8 +202,8 @@ public void testElasticsearchWithSchema(TestContainer container)
     @TestTemplate
     @DisabledOnContainer(
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-elasticsearch-e2e/src/test/java/org/apache/seatunnel/e2e/connector/elasticsearch/ElasticsearchIT.java]: Please delete this annotation `@DisabledOnContainer`

**Comment by Hisoka-X** [comment]
cc @TyrantLucifer

**Pull request closed without merging**